packages/instant/src/components/erc20_token_selector.tsx

```diff
@@ -85,7 +85,7 @@ class TokenSelectorRow extends React.Component<TokenSelectorRowProps> {
                 <Container marginLeft="5px">
                     <Flex justify="flex-start">
                         <Container marginRight="10px">
-                            <Circle diameter={30} fillColor={token.metaData.primaryColor}>
+                            <Circle diameter={30} rawColor={token.metaData.primaryColor}>
```

Comment:
Yea we need the `Circle` component to take arbitrary hexes as a color since it also powers the token fallback UI in the token selector, and those colors can be arbitrary.